content/03.ml-concepts.md

```diff
@@ -1,24 +1,29 @@
-## Tip 1: Concepts that apply to machine learning also apply to deep learning {#concepts}
-Deep learning is a distinct subfield of machine learning, but it is still a subfield.
-DL has proven to be an extremely powerful paradigm capable of outperforming “traditional” machine learning approaches in certain contexts, but it is not immune to the many limitations inherent to machine learning.
-Many best practices for machine learning also apply to deep learning.
-Like all computational methods, deep learning should be applied in a systematic manner that is reproducible and rigorously tested.
+## Tip 1: Adhere to machine learning best practices for rigorous model evaluation {#concepts}
 
-Those developing deep learning models should select data that are relevant to the problem at hand; non-salient data can hamper performance or lead to spurious conclusions.
-Biases in testing data can also unduly influence measures of model performance, and it may be difficult to directly identify confounders from the model.
-Investigators should consider the extent to which the outcome of interest is likely to be predictable from the input data and begin by throughly inspecting the input data.
-Suppose that there are robust heritability estimates for a phenotype that suggest that the genetic contribution is modest but a deep learning model predicts the phenotype with very high accuracy.
+
+In recent years, deep learning has proven to be an extremely powerful paradigm capable of outperforming "traditional" or "conventional" machine learning approaches.
+The recent success behind applying deep learning is partly owed to the fact that DL algorithms are less dependent on manual feature extraction and are capable of achieving high predictive performance when operating on raw-data, such as images and text directly.
+In contrast, traditional machine learning approaches often require manual feature extraction to produce acceptable results. 
+However, the ability of deep learning to perform implicit feature extraction can be regarded as a double-edged sword.
+For instance, those developing deep learning models not only need to ensure that the data contains information relevant to the problem at hand but is also free of systematic biases.
+As deep learning models are capable of extracting and learning from complex patterns in a training dataset, DL models are also very susceptible to systematic noise and non-salient information, which can not only hamper performance but also lead to spurious conclusions, making proper model evaluation an ever so important topic.
+
+While deep learning models require careful evaluation, good news is that since deep learning is still a subfield of machine learning, the majority of best practices that were developed for machine learning over the last decades also apply to DL.
```

Comment:
I would move this idea up earlier in the tip. The impetus of this tip is that "deep learning is still a subfield of machine learning", and IMO that should established at the start of tip 1. This sentence also has some grammatical errors.